src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint.py

```diff
@@ -315,12 +315,9 @@ def __call__(
         latents = init_latents
 
         t_start = max(num_inference_steps - init_timestep + offset, 0)
+        timesteps = self.scheduler.timesteps.copy()[t_start:]
```

Comment:
why do we need a `copy()` here?

Reply:
Oh, It is just what I forgot to remove and is a needless copy.
I removed it at dca59b8, but also works fine.